### PR TITLE
CHIPS Security Test for LTPA

### PR DESCRIPTION
**Pull request by volosied**
Test for  #27983

We have one samesite security test in this FAT, so I reused it for partitioning.

### CODE REVIEW

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_sameSiteSecurity/server.xml [hunk 1]
```diff
@@ -12,32 +12,40 @@
...
         <feature>appSecurity-2.0</feature>
         <feature>jsp-2.3</feature>
     </featureManager>
 
-    <keyStore id="defaultKeyStore" password="{xor}EzY9Oi0rJg=="/> <!-- pwd: Liberty, expires 1/4/2099 -->
+    <httpEndpoint id="defaultHttpEndpoint"
+        host="*"
+        httpPort="${bvt.prop.HTTP_default}"
+        httpsPort="${bvt.prop.HTTP_default.secure}">
+        <samesite lax="*" />
```
Comment (volosied) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_sameSiteSecurity/server.xml:27]: Server.xml was different fro the javadoc: 

https://github.com/OpenLiberty/open-liberty/blob/16197b274fe83d51a34062c47cea00e1bc4e80a4/dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCSameSiteCookieAttributeSecurityTest.java#L79C1-L81C54

**Comment by volosied** [comment]
#build

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedCookieAttributeSecurityTest.java [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+            nvps.add(new BasicNameValuePair("j_username", userName));
+            nvps.add(new BasicNameValuePair("j_password", password));
+            postMethod.setEntity(new UrlEncodedFormEntity(nvps));
+
+            try (final CloseableHttpResponse response = client.execute(postMethod)) {
+                LOG.info("Post Method response code: " + response.getCode());
+                assertEquals("The expected form login status code was not returned: ", 302,
+                             response.getCode());
+
+                // Verify that the LtpaToken2 cookie has SameSite=Strict
```
Comment (volosied) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedCookieAttributeSecurityTest.java]: Update comment - None and Partitioned

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedCookieAttributeSecurityTest.java [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+                    LOG.info("Header Value: " + cookieHeaderValue);
+
+                    if (cookieHeaderValue.startsWith("LtpaToken2")) {
+                        if (cookieHeaderValue.contains("SameSite=None") && cookieHeaderValue.contains("Partitioned")) {
+                            headerFound = true;
+                        }
+                    }
+                }
+
+                assertTrue("The LtpaToken2 Set-Cookie header did not contain SameSite=Strict.", headerFound);
```
Comment (volosied) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedCookieAttributeSecurityTest.java]: Update text

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedCookieAttributeSecurityTest.java [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCPartitionedCookieAttributeSecurityTest.java]: No need for the contributors in the license header any longer.

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_partitionedSecurity/bootstrap.properties [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_partitionedSecurity/bootstrap.properties:9]: No need for the contributors

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_partitionedSecurity/server.xml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    Copyright (c) 2024 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_partitionedSecurity/server.xml]: NO need for the contributors

**Comment by volosied** [comment]
#libby

**Review by pnicolucci — APPROVED** [review]

**Pull request merged**